frontend/src/components/regions/AgileBoard/AgileBoard.test.js

```diff
@@ -0,0 +1,57 @@
+import  { boardFromCards }  from "../AgileBoard";
+import { cards, latestCalls} from "./mockedData";
+
+test("boardFromCards function returns correct total length 5 of the array", () => {
+   let boardFromCard  = boardFromCards({ cards, latestCalls });
```

Comment:
I know that these are just tests. Is there a specific reason they are declared with let instead of const? `const` seems more fiting.

Reply:
const should be used here. Remember that we hold the learners to a high standard with this sort of thing.

Reply:
I will change to const